x-pack/solutions/security/plugins/security_solution/public/flyout/document_details/right/components/visualizations_section.tsx

```diff
@@ -49,6 +52,12 @@ export const VisualizationsSection = memo(() => {
   const shouldShowGraphPreview =
     visualizationInFlyoutEnabled && isGraphFeatureEnabled && hasGraphRepresentation;
 
+  if (shouldShowGraphPreview) {
+    telemetry.reportEvent(DocumentEventTypes.DetailsGraphPreviewVisible, {
+      location: scopeId,
+    });
+  }
```

Comment:
We can report this event inside `<GraphPreviewContainer />` instead, so it will be reported even if the preview is shown outside of this file context